Cron/PendingOrdersUpdate.php

```diff
@@ -59,7 +59,8 @@ public function execute(): static
             }
             $amwalOrderData = $this->getAmwalOrderData->execute($amwalOrderId);
             if (!$amwalOrderData || $amwalOrderData['status'] !== 'success') {
-                continue;
+                $order->setState(Order::STATE_CANCELED);
```

Comment:
This might be an issue if a user is in the payment process (order pending payment) and the cron is triggered.

Maybe we should instead check for a specific status to cancel the order?

Reply:
We cover it by getting the old orders by time periods, so this scenario will never happen